service/cloudfront/sign/sign_cookie_test.go

```diff
@@ -0,0 +1,85 @@
+package sign
+
+import (
+	"crypto/rsa"
+	"testing"
+	"time"
+)
+
+const (
+	policyName    = "CloudFront-Policy"
+	signatureName = "CloudFront-Signature"
+	keyName       = "CloudFront-Key-Pair-Id"
+)
+
+func TestSignCookie(t *testing.T) {
```

Comment:
I'd suggest breaking these tests out into multiple Test functions. This will make tracking down future bugs easier and tests would be more focused.